tests_metricflow/snapshots/test_offset_metrics.py/str/DuckDB/test_custom_offset_window_with_base_grain__query_output.txt

```diff
@@ -0,0 +1,21 @@
+test_name: test_custom_offset_window_with_base_grain
+test_filename: test_offset_metrics.py
+docstring:
+  Gives a side by side comparison of bookings and bookings_offset_one_martian_day.
+---
+metric_time__martian_day    metric_time__day     bookings    bookings_offset_one_martian_day
```

Comment:
* With how `martian_day` is defined, it's a little hard to check these snapshots for correctness. 
* A more straightforward mapping would help e.g. martian days are always 10 years before.
* Also time on Mars is represented differently externally, so a different name would help.